docs/sdks/server-sdks/node.md

```diff
@@ -84,6 +55,9 @@ const assignmentLogger: IAssignmentLogger = {
 };
 ```
 
+
+
+
 The SDK will invoke the `logAssignment` function with an `assignment` object that contains the following fields:
 
 | Field                     | Description                                                                                                              | Example                             |
```

Comment:
This table feels a bit like clutter. Can we just mention that Eppo's SDK creates an assignment analytic event and then point the reader either to the example section below, or our [page on event logging](https://docs.geteppo.com/sdks/event-logging)?

Reply:
moved to the bottom of the page